app/packs/stylesheets/decidim/extra_user_fields/_heatmap.scss

```
@@ -0,0 +1,79 @@
+  --heatmap-total-1: hsl(215, 53%, 87%);
+  --heatmap-total-2: hsl(215, 57%, 77%);
+  --heatmap-total-3: hsl(215, 61%, 67%);
+  --heatmap-total-4: hsl(215, 63%, 57%);
+  --heatmap-total-5: hsl(215, 65%, 50%);
+  --heatmap-non-specified: hsl(0, 0%, 80%);
+
+  @apply flex items-center gap-3 mt-3 text-sm text-gray-2;
+
+  &__gradient {
+    @apply flex h-4 rounded overflow-hidden;
+    width: 160px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)